### PR TITLE
Replace retirement tool hero with intro text and image

### PR DESCRIPTION
**Pull request by virginiacc**
Update style of retirement tool intro based on its new place in the page hierarchy after addition of retirement portal page

## Additions

- New Spanish and English circular images
- Breadcrumbs & associated less file

## Changes

- Replace hero with intro text and circular image
- Update less for intro

## Testing

- Check out branch
- Run `gulp styles`
- Navigate to English and Spanish versions of retirement tool and verify that intro matches screenshots

## Screenshots

screencapture-localhost-8000-retirement-before-you-claim-1505235907238
screencapture-localhost-8000-retirement-before-you-claim-1505236399454

screencapture-localhost-8000-retirement-before-you-claim-es-1505235918420
screencapture-localhost-8000-retirement-before-you-claim-es-1505236382341

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices 
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: retirement_api/urls.py [hunk 1]
```diff
@@ -41,4 +41,4 @@
         url(r'^retirement/retirement-api/estimator/(?P<dob>[^/]+)/(?P<income>\d+)/es/$',
             'retirement_api.views.estimator',
             {'language': 'es'}, name='estimator_es'),
-        ]
+        ]
```
Comment (marteki) [inline retirement_api/urls.py]: Missing newline.

File: src/css/breadcrumbs.less [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    &_link {
+        padding-left: 4px;
+
+        &:not( :last-child ):after {
+            padding-left: 4px;
+            color: @text;
+            content: ' /';
+        }
+    }
+}
```
Comment (marteki) [inline src/css/breadcrumbs.less]: Missing newline.

File: src/css/cf-enhancements.less [hunk 3]
```diff
@@ -529,3 +529,9 @@ input[type=range] {
     }
   }
 }
+
+.wrapper__match-content {
+    .respond-to-min(@bp-sm-min, {
+        max-width: @grid_wrapper-width - ( @grid_gutter-width * 2 ) !important;
+    });
+}
```
Comment (marteki) [inline src/css/cf-enhancements.less]: Missing newline.

File: src/css/claiming.less [hunk 4]
```diff
@@ -84,53 +84,58 @@ form {
...
 
-#claiming-social-security .claiming-hero {
-  background-color: #FFCE8D;
+[data-lang="en"] #claiming-social-security .intro  {
+    margin-top: 15px;
 }
 
-#claiming-social-security .claiming-hero .hero_wrapper {
-  position: relative;
+@media (min-width: 850px) {
```
Comment (marteki) [inline src/css/claiming.less]: There's a widow-orphan situation in the hero text for screen sizes from 850px to 891px.

screen shot 2017-09-12 at 2 38 10 pm

**Comment by virginiacc** [comment]
Thanks for reviewing, @marteki! The newlines should be fixed in https://github.com/cfpb/retirement/pull/201/commits/ca8eca5979534628e10134bba1f2b925ed431176. I'll check with the team about the headline orphan -- maybe we could drop the image around 900px to leave more space for the text.

**Comment by marteki** [comment]
Sounds good. In my view, the headline orphan doesn't need to hold up this PR merge.

**Comment by virginiacc** [comment]
Thanks, @marteki!

**Pull request merged**